src/draw_target.rs

```diff
@@ -485,6 +485,22 @@ impl LineType {
             LineType::Empty => "",
         }
     }
+
+    fn console_width(&self) -> usize {
```

Comment:
Nit: `console_width()` should appear after `wrapped_height()`, since the latter depends on the former.